docs/Gemfile.lock

```diff
@@ -14,14 +14,14 @@ GEM
       execjs
     coffee-script-source (1.11.1)
     colorator (1.1.0)
-    commonmarker (0.23.4)
+    commonmarker (0.17.13)
```

Comment:
Ah yes, @AnnaKornfeldSimpson I thought that this drop in commonmarker was suspicious.